### ISSUE TITLE
[missing-page]

### ISSUE THREAD

**Issue by jsteenkamp** [aws-amplify/docs#2045]
**Page**: [`https://docs.amplify.aws/lib/storage/autotrack/q/platform/js`](https://docs.amplify.aws/lib/storage/autotrack/q/platform/js)

**Feedback**:

**Comment by mauerbac** [comment]
hi @jsteenkamp - 

Can you explain a bit more? That page is able to load for me.

**Comment by jsteenkamp** [comment]
I simply reported by clicking link on the 404 page. Tested now and same result as per image in Chrome and Firefox, worked in Microsoft Edge (Chromium version) and Safari. I wonder if it is due to cache/manifest as I use Chrome and Firefox mostly. All other pages in docs seem fine.

<img width="1436" alt="Screen Shot 2020-07-07 at 2 22 26 PM" src="https://user-images.githubusercontent.com/343072/86691871-5b088d00-c05d-11ea-9d3d-9112cd36256c.png">
